jbpm-human-task/jbpm-human-task-jpa/pom.xml

```diff
@@ -75,6 +75,11 @@
     </dependency>
 
     <!-- TEST -->
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-test-util</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
It's here: https://github.com/kiegroup/jbpm/pull/898/commits/e1683e403aff2063ab5026e25039b0e2d4486ee7